Index: src/test/java/com/cydeo/pages/AutomationHomePage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.cydeo.pages;\n\nimport com.cydeo.utilities.Driver;\nimport org.openqa.selenium.WebElement;\nimport org.openqa.selenium.support.FindBy;\nimport org.openqa.selenium.support.PageFactory;\n\npublic class AutomationHomePage {\n\n    public AutomationHomePage(){\n\n        PageFactory.initElements(Driver.getDriver(),this);\n    }\n\n    @FindBy(xpath = \"//a[@href=\\\"/login\\\"]\")\n    public WebElement signupLoginButton;\n\n    @FindBy(xpath = \"//b[.='Kasim Safa']\")\n    public WebElement loggedInUserName;\n\n    @FindBy(xpath = \"//a[.=' Delete Account']\")\n    public WebElement deleteAccountButton;\n\n\n    @FindBy(xpath = \"//i[@class='fa fa-user']\")\n    public WebElement loggedInUser;\n\n\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/com/cydeo/pages/AutomationHomePage.java b/src/test/java/com/cydeo/pages/AutomationHomePage.java
--- a/src/test/java/com/cydeo/pages/AutomationHomePage.java	(revision fa50022739e3aeffb5b22cfc4d4d68a4c4e4377b)
+++ b/src/test/java/com/cydeo/pages/AutomationHomePage.java	(date 1689090636475)
@@ -21,7 +21,6 @@
     @FindBy(xpath = "//a[.=' Delete Account']")
     public WebElement deleteAccountButton;
 
-
     @FindBy(xpath = "//i[@class='fa fa-user']")
     public WebElement loggedInUser;
 
